src/publishPackages/publish.proj

```diff
@@ -53,33 +56,95 @@
 
   <Target Name="PreparePublish" DependsOnTargets="$(CreateDependsOn)">
     <MakeDir Directories="$(ManifestDir)" />
-    <PropertyGroup>
-      <ExpectedFeedUrl>https://$(AzureAccountName).blob.core.windows.net/$(ContainerName)/index.json</ExpectedFeedUrl>
-      <AccountKey>$(AzureAccessToken)</AccountKey>
-    </PropertyGroup>
+    <MakeDir Directories="$(ArtifactsTempWorkDir)" />
+    <!-- Include other build data attributes : (Src) https://github.com/dotnet/arcade/blob/master/src/Microsoft.DotNet.Arcade.Sdk/tools/Publish.proj#L167 -->
+    <ItemGroup>
+      <NewManifestBuildData Include="InitialAssetsLocation=$(SYSTEM_TEAMFOUNDATIONCOLLECTIONURI)$(SYSTEM_TEAMPROJECT)/_apis/build/builds/$(BUILD_BUILDID)/artifacts" />
+      <NewManifestBuildData Include="AzureDevOpsBuildId=$(BUILD_BUILDID)" />
+      <NewManifestBuildData Include="AzureDevOpsBuildDefinitionId=$(SYSTEM_DEFINITIONID)" />
+      <NewManifestBuildData Include="AzureDevOpsAccount=$(AzureAccountName)" />
```

Comment:
This is incorrect. This should be the Azure DevOps account where the build lives, IE dnceng. Not a storage account, which is resolving to dotnetcli in this case

Reply:
@adaggarwal I talked with @riarenas about this. Feel free to ping me on Teams if you have any question. 

Looks like you're receiving $(AzureAccountName) as a parameter from the command line. You don't need that. Take a look how Arcade SDK Publish.proj computes $(AzureAccountName).